ml-testing-toolkit/values.yaml

```diff
@@ -4,47 +4,111 @@ global: {}
 
 ml-testing-toolkit-backend:
   enabled: true
+  # ingress:
+  #   enabled: true
+  #   hosts:
+  #     specApi:
+  #       host: testing-toolkit-specapi.localdomain
+  #       port: 5000
+  #       paths: ['/']
+  #     adminApi:
+  #       host: testing-toolkit.localdomain
+  #       port: 5050
+  #       paths: ['/api/', '/socket.io/']
   config:
     system_config.json:
       HOSTING_ENABLED: false
       OAUTH:
         AUTH_ENABLED: false
-      CONNECTION_MANAGER_AUTH_ENABLED: false
+      CONNECTION_MANAGER:
```

Comment:
Is this not a breaking change?

I.e. requires a major version bump to the helm chart?

Reply:
No Miguel, that param is related to hosted mode and left in old syntax from long time. So we can consider this as a minor fix and don't need major version bump.

Reply:
May be we can change minor version of TTK helm chart